ClassNotFoundException when running analysers

The latest release of the plugin 1.2.0 has a bug when running the analysers (see below).
This is caused by updated the Sling Feature Analyser to the latest version in https://github.com/adobe/aemanalyser-maven-plugin/commit/68dd10180ba7f0d175fc5680a9b238656bf5de0c

Exception in thread "main" java.util.ServiceConfigurationError: org.apache.sling.feature.analyser.task.AnalyserTask: org.apache.sling.feature.analyser.task.impl.CheckContentPackages Unable to get public no-arg constructor
        at java.base/java.util.ServiceLoader.fail(ServiceLoader.java:582)
        at java.base/java.util.ServiceLoader.getConstructor(ServiceLoader.java:673)
        at java.base/java.util.ServiceLoader$LazyClassPathLookupIterator.hasNextService(ServiceLoader.java:1233)
        at java.base/java.util.ServiceLoader$LazyClassPathLookupIterator.hasNext(ServiceLoader.java:1265)
        at java.base/java.util.ServiceLoader$2.hasNext(ServiceLoader.java:1300)
        at java.base/java.util.ServiceLoader$3.hasNext(ServiceLoader.java:1385)
        at org.apache.sling.feature.analyser.task.AnalyzerTaskProvider.getTasksByIds(AnalyzerTaskProvider.java:62)
        at org.apache.sling.feature.analyser.Analyser.<init>(Analyser.java:140)
        at com.adobe.aem.analyser.AemAnalyser.createAnalyser(AemAnalyser.java:188)
        at com.adobe.aem.analyser.AemAnalyser.analyse(AemAnalyser.java:206)
        at com.adobe.aem.analyser.mojos.AemAnalyseMojo.analyseFeatures(AemAnalyseMojo.java:351)
        at com.adobe.aem.analyser.mojos.AemAnalyseMojo.execute(AemAnalyseMojo.java:238)
        at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo(DefaultBuildPluginManager.java:137)
        at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:210)
        at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:156)
        at org.apache.maven.lifecycle.internal.MojoExecutor.execute(MojoExecutor.java:148)
        at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:117)
        at org.apache.maven.lifecycle.internal.LifecycleModuleBuilder.buildProject(LifecycleModuleBuilder.java:81)
        at org.apache.maven.lifecycle.internal.builder.singlethreaded.SingleThreadedBuilder.build(SingleThreadedBuilder.java:56)
        at org.apache.maven.lifecycle.internal.LifecycleStarter.execute(LifecycleStarter.java:128)
        at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:305)
        at org.apache.maven.DefaultMaven.doExecute(DefaultMaven.java:192)
        at org.apache.maven.DefaultMaven.execute(DefaultMaven.java:105)
        at org.apache.maven.cli.MavenCli.execute(MavenCli.java:957)
        at org.apache.maven.cli.MavenCli.doMain(MavenCli.java:289)
        at org.apache.maven.cli.MavenCli.main(MavenCli.java:193)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.base/java.lang.reflect.Method.invoke(Method.java:566)
        at org.codehaus.plexus.classworlds.launcher.Launcher.launchEnhanced(Launcher.java:282)
        at org.codehaus.plexus.classworlds.launcher.Launcher.launch(Launcher.java:225)
        at org.codehaus.plexus.classworlds.launcher.Launcher.mainWithExitCode(Launcher.java:406)
        at org.codehaus.plexus.classworlds.launcher.Launcher.main(Launcher.java:347)
Caused by: java.lang.NoClassDefFoundError: org/apache/jackrabbit/vault/validation/spi/ValidationMessageSeverity
        at java.base/java.lang.Class.getDeclaredConstructors0(Native Method)
        at java.base/java.lang.Class.privateGetDeclaredConstructors(Class.java:3137)
        at java.base/java.lang.Class.getConstructor0(Class.java:3342)
        at java.base/java.lang.Class.getConstructor(Class.java:2151)
        at java.base/java.util.ServiceLoader$1.run(ServiceLoader.java:660)
        at java.base/java.util.ServiceLoader$1.run(ServiceLoader.java:657)
        at java.base/java.security.AccessController.doPrivileged(Native Method)
        at java.base/java.util.ServiceLoader.getConstructor(ServiceLoader.java:668)
        ... 32 more
Caused by: java.lang.ClassNotFoundException: org.apache.jackrabbit.vault.validation.spi.ValidationMessageSeverity
        at org.codehaus.plexus.classworlds.strategy.SelfFirstStrategy.loadClass(SelfFirstStrategy.java:50)
        at org.codehaus.plexus.classworlds.realm.ClassRealm.unsynchronizedLoadClass(ClassRealm.java:271)
        at org.codehaus.plexus.classworlds.realm.ClassRealm.loadClass(ClassRealm.java:247)
        at org.codehaus.plexus.classworlds.realm.ClassRealm.loadClass(ClassRealm.java:239)
        ... 40 more

--- a/aemanalyser-maven-plugin/pom.xml
+++ b/aemanalyser-maven-plugin/pom.xml
@@ -156,7 +156,12 @@ governing permissions and limitations under the License.
     <dependency>
       <groupId>org.apache.jackrabbit.vault</groupId>
       <artifactId>org.apache.jackrabbit.vault</artifactId>
-      <version>3.4.10</version>
+      <version>3.5.4</version>
+    </dependency>
+    <dependency>
+        <groupId>org.apache.jackrabbit.vault</groupId>
+        <artifactId>vault-validation</artifactId>
+        <version>3.5.4</version>
     </dependency>
     <dependency>
       <groupId>org.apache.sling</groupId>
@@ -181,17 +186,17 @@ governing permissions and limitations under the License.
     <dependency>
       <groupId>org.apache.jackrabbit</groupId>
       <artifactId>jackrabbit-api</artifactId>
-      <version>2.18.4</version>
+      <version>2.19.1</version>
     </dependency>
     <dependency>
       <groupId>org.apache.jackrabbit</groupId>
       <artifactId>jackrabbit-jcr-commons</artifactId>
-      <version>2.18.4</version>
+      <version>2.19.1</version>
     </dependency>
     <dependency>
       <groupId>org.apache.jackrabbit</groupId>
       <artifactId>jackrabbit-spi-commons</artifactId>
-      <version>2.18.4</version>
+      <version>2.19.1</version>
     </dependency>
     <dependency>
       <groupId>org.apache.sling</groupId>

